src/components/buttons/_buttons.scss

```diff
@@ -138,43 +138,38 @@ $largeButtonSize: componentSize(l);
   }
 
   &--solid-inverted {
-    @include buttonHover($gray-10, $white, 80%);
+    @include buttonHover($gray-20, $white, 80%);
     background-color: $white;
     color: $black;
   }
 
-  &--solid-blue {
-    @include buttonHover($blue-50, $blue-40, 80%);
-    background-color: $blue-40;
+  &--solid-indigo {
+    @include buttonHover($indigo-60, $indigo-50, 80%);
+    background-color: $indigo-50;
   }
 
-  &--solid-mint {
-    @include buttonHover($green-50, $green-40, 80%);
-    background-color: $green-40;
+  &--solid-indigo-inverted {
+    @include buttonHover($gray-20, $white, 80%);
+    background-color: $white;
+    color: $indigo-50;
   }
 
   &--solid-light {
-    @include buttonHover($gray-40, $gray-20, 12%);
+    @include buttonHover($gray-50, $gray-20, 12%);
```

Comment:
@clxandstuff could we maybe parameterise these percent values?

Reply:
Do you mean to store it in variable? 😃